src/Languages/Base/Injections/GutterInjection.php

```diff
@@ -62,7 +63,7 @@ public function parse(string $content, Highlighter $highlighter): ParsedInjectio
             $gutterClass = 'hl-gutter' . ($hasClasses ? ' ' . $hasClasses : '');
 
             $lines[$i] = sprintf(
-                Escape::tokens('<span class="%s">%s</span> %s'),
+                Escape::tokens('<span class="%s">%s</span>'.($highlighter->getTheme() instanceof TerminalTheme ? ' ' : '').'%s'),
```

Comment:
I think it would be cleaner to write it like so:

```php
sprintf(
    Escape::tokens('<span class="%s">%s</span>%s%s'),
    $gutterClass,
    str_pad(…),
    $highlighter->getTheme() instanceof TerminalTheme ? ' ' : '',
    $line
)
```

Could you change that?

Reply:
Good point! New commit added.